internal/cli/users_roles.go

```diff
@@ -141,7 +142,67 @@ func addUserRolesCmd(cli *cli) *cobra.Command {
 	return cmd
 }
 
-func (cli *cli) pickUserRoles(inputs *userRolesInput) error {
+func removeUserRolesCmd(cli *cli) *cobra.Command {
+	var inputs userRolesInput
+
+	cmd := &cobra.Command{
+		Use:     "remove",
+		Aliases: []string{"rm"},
+		Args:    cobra.MaximumNArgs(1),
+		Short:   "Remove roles from a user",
+		Long:    "Remove existing roles from a user.",
+		Example: `  auth0 users roles remove <user-id>
+  auth0 users roles remove <user-id> --roles <role-id1,role-id2>
+  auth0 users roles rm <user-id> -r "rol_1eKJp3jV04SiU04h,rol_2eKJp3jV04SiU04h" --json`,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if len(args) == 0 {
+				if err := userID.Ask(cmd, &inputs.ID); err != nil {
+					return err
+				}
+			} else {
+				inputs.ID = args[0]
+			}
+
+			if len(inputs.Roles) == 0 {
+				if err := cli.pickUserRolesToRemove(&inputs); err != nil {
+					return err
+				}
+			}
+
+			var rolesToRemove []*management.Role
+			for _, roleID := range inputs.Roles {
+				rolesToRemove = append(rolesToRemove, &management.Role{
+					ID: auth0.String(roleID),
+				})
+			}
+
+			if err := ansi.Waiting(func() (err error) {
+				return cli.api.User.RemoveRoles(inputs.ID, rolesToRemove)
```

Comment:
Should check to see if length of `rolesToRemove` is non-zero, otherwise will result in a 400.